src/pages/Restaurant/Restaurant.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './Restaurant.scss';
+
+export default function Restaurant(props) {
+  const { Res_id, Res_img, Res_name, Rates, Reviews } = props;
```

Comment:
대영님! 구조분해할당 잘 해주셨네요! 
해당 컴포넌트의 매개변수 자리에서 바로 구조분해할당을 해주어도 좋을 것 같습니다!
더불어서 props명칭이 대문자로 시작하고 _가 들어가는데.. 어떤 네이밍 형식일까요...?!

```suggestion
export default function Restaurant({ Res_id, Res_img, Res_name, Rates, Reviews } ) {
```